pytest_timeout.py

```diff
@@ -381,8 +386,8 @@ def timeout_timer(item, timeout):
     try:
         capman = item.config.pluginmanager.getplugin("capturemanager")
         if capman:
-            pytest_version = StrictVersion(pytest.__version__)
-            if pytest_version >= StrictVersion("3.7.3"):
+            pytest_version = LooseVersion(pytest.__version__)
+            if pytest_version >= LooseVersion("3.7.3"):
```

Comment:
I'd probably just have bumped the pytest dependency requirement to past 3.7.3 and removed this check entirley instead of fixing this.  This is pretty old by now.